src/ManifestStrategy/GlossaryManifestStrategy.php

```diff
@@ -36,6 +36,7 @@ public function getType(): string
      */
     public function apply(array $manifest, string $moduleName, InputOutputInterface $inputOutput, bool $isDry): bool
     {
+        echo 'TEST';
```

Comment:
Those things should be tested locally, but not commited.

Reply:
Locally all works fine, problem only on stage environment

Reply:
What staging?
We only have testing?

Reply:
Yea i am testing on linux

Reply:
How, i can reproduce testing environment on local environment?